### PR TITLE
Add script pktgen_perf.sh and update perf.conf file for performance

### PR DESCRIPTION
**Pull request by quanwenli**
pktgen_perf test case.

Signed-off-by: Wenli Quan <wquan@redhat.com>

ID: 1470602

### CODE REVIEW

**Comment by quanwenli** [comment]
Please review it with https://github.com/autotest/tp-qemu/pull/1156 together.

File: shared/scripts/pktgen_perf.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    PGDEV=/proc/net/pktgen/$dev
+
+    pgset "queue_map_min $i"
+    pgset "queue_map_max $i"
+    pgset "count 0"
+    pgset "min_pkt_size $SIZE"
+    pgset "max_pkt_size $SIZE"
+    echo $DST | grep ":"
+    if [ $? -eq 0 ]
+	then
```
Comment (balamuruhans) [inline shared/scripts/pktgen_perf.sh]: trailing white spaces can be removed.

File: shared/scripts/pktgen_perf.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    pgset "queue_map_min $i"
+    pgset "queue_map_max $i"
+    pgset "count 0"
+    pgset "min_pkt_size $SIZE"
+    pgset "max_pkt_size $SIZE"
+    echo $DST | grep ":"
+    if [ $? -eq 0 ]
+	then 
+        pgset "dst_mac $DST"
+    else
```
Comment (balamuruhans) [inline shared/scripts/pktgen_perf.sh]: same here

File: shared/scripts/pktgen_perf.sh [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    pgset "count 0"
+    pgset "min_pkt_size $SIZE"
+    pgset "max_pkt_size $SIZE"
+    echo $DST | grep ":"
+    if [ $? -eq 0 ]
+	then 
+        pgset "dst_mac $DST"
+    else 
+     	pgset "dst $DST"
+    fi
```
Comment (balamuruhans) [inline shared/scripts/pktgen_perf.sh]: trailing white space.

File: shared/scripts/pktgen_perf.sh [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    pgset "queue_map_max $i"
+    pgset "count 0"
+    pgset "min_pkt_size $SIZE"
+    pgset "max_pkt_size $SIZE"
+    echo $DST | grep ":"
+    if [ $? -eq 0 ]
+	then 
+        pgset "dst_mac $DST"
+    else 
+     	pgset "dst $DST"
```
Comment (balamuruhans) [inline shared/scripts/pktgen_perf.sh]: space and tab in the indentation.

File: shared/scripts/pktgen_perf.sh (hunk 3 above)
Comment (quanwenli) [reply shared/scripts/pktgen_perf.sh]: @balamuruhans I have update it, thanks for your review.

File: shared/scripts/pktgen_perf.sh [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    if [ "$result" = "" ]; then
+         cat $PGDEV | fgrep Result:
+    fi
+}
+
+function pg() {
+    echo inject > $PGDEV
+    cat $PGDEV
+}
+
```
Comment (YanhuiMa) [inline shared/scripts/pktgen_perf.sh]: This function isn't called in the script.

Comment (quanwenli) [reply shared/scripts/pktgen_perf.sh]: yes, it's not be called. the interest thing is all the examples I could found from web are all included function pg, example https://github.com/danieltt/pktgen/tree/master/examples , and also pktgen.sh in avocado-vt, so I think let's just keep this in this script. thx

Comment (YanhuiMa) [reply shared/scripts/pktgen_perf.sh]: If there is no error when remove this function, I think it's better to remove it.

**Comment by quanwenli** [comment]
update it as a middle way: comment out function pg.

**Comment by YanhuiMa** [comment]
(1/1) smp_32.8192m.repeat1.run_test.Host_RHEL.m7.u5.raw.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.myprovider.pktgen_perf: PASS (139.76 s)
Acked-by: Yanhui Ma yama@redhat.com

File: shared/scripts/pktgen_perf.sh [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    result=`cat $PGDEV | fgrep "Result: OK:"`
+    if [ "$result" = "" ]; then
+         cat $PGDEV | fgrep Result:
+    fi
+}
+
+#function pg() {
+#    echo inject > $PGDEV
+#    cat $PGDEV
+#}
```
Comment (luckyh) [inline shared/scripts/pktgen_perf.sh]: Please remove these lines if they are useless.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**